morpholopy/galaxy_data.py

```diff
@@ -807,22 +807,25 @@ def process_galaxy(args) -> Tuple[int, NDArray[data_fields], Union[None, Dict]]:
         images[f"ZZZ - Galaxy {galaxy_index:08d}"].update(
             galaxy_images["Visualisation"]
         )
+        gallery_images = galaxy_images["Gallery"]
     
-        # individual KS plots
-        galaxy_data.compute_medians()
-        KS_images = plot_KS_relations(
-            output_path,
-            observational_data_path,
-            [f"Galaxy {galaxy_index}"],
-            [galaxy_data],
-            prefix=f"galaxy_{index:03d}_",
-            always_plot_scatter=True,
-            plot_integrated_quantities=False,
-        )
-        images[f"ZZZ - Galaxy {galaxy_index:08d}"].update(
-            KS_images["Combined surface densities"]
-        )
+        if make_individual_KS_plots:
+            # individual KS plots
+            galaxy_data.compute_medians()
+            KS_images = plot_KS_relations(
+                output_path,
+                observational_data_path,
+                [f"Galaxy {galaxy_index}"],
+                [galaxy_data],
+                prefix=f"galaxy_{index:03d}_",
+                always_plot_scatter=True,
+                plot_integrated_quantities=False,
+            )
+            images[f"ZZZ - Galaxy {galaxy_index:08d}"].update(
+                KS_images["Combined surface densities"]
+            )
     else:
         images = None
+        gallery_images = None
 
-    return index, galaxy_data, images
+    return index, galaxy_data, images, gallery_images
```

Comment:
Since this function now returns a new object `gallery_images`, the signature (for type hints) and comment to this function (`process_galaxy()`) need to be updated to reflect this change.

Reply:
Added both